.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -22,3 +22,90 @@ Instructions MAY be by reference.
+  - [ ] How to run tests
+  - [ ] Where to file challenges
+
+8. Compatibility certification request
+  - [ ] Request follows [template](https://github.com/jakartaee/specification-committee/blob/master/compatibility-certification-request.md)
+  - [ ] SHA-256 fingerprint matches staged TCK zip file
+  - [ ] Request issue has `certification` label.
+
+9. TCK results summary
+  - [ ] Page is hosted by Compatible Implementation project
+  - [ ] Includes all information from certification request
+  - [ ] Summary includes number of tests passed, failed, errors
+  - [ ] SHA-256 fingerprint matches staged TCK zip file on cert request
+
+10. If a Release Review is required, the specification project team contacts the EMO to initiate the review by sending an email to emo@eclipse.org.
+    (A Release Review is not required if the current release is a Service Release based on a previously successful Major or Minor
+    release as indicated by a release record on the project's Releases page, e.g., the [Jakarta Servlet releases page](https://projects.eclipse.org/projects/ee4j.servlet/reviews).)
+    - [ ] An issue will be created by the EMO to track the release review.
+    - [ ] The specification project team requests approval for the release from the PMC by sending an email to ee4j-pmc@eclipse.org.
+    - [ ] The specification project team then delivers an IP Log to the IP Team for their review as described in https://www.eclipse.org/projects/handbook/#pmi-commands-iplog.
+
+11. Update Jakarta EE API jar
+  - [ ] Update the Jakarta EE API jar by submitting a PR to the jakartaee-api project that updates the version number of your API jar file.
+
+### Mentor Final Tasks after a Successful Release Review
+
+ - [ ] adds this final checklist to the main PR.
+ - [ ] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
+ - [ ] calculates the staged EFTL TCK signature and promotes it to the committee download area
+  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
+ - [ ] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
+ - [ ] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
+ - [ ] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
+ - [ ] creates an issue in the specification project that includes a checklist for the specification project team:
+(an example can be found [here](https://github.com/jakartaee/authorization/issues/166).)
```

Comment:
I think it would be more helpful to add the checklist in raw format here for the mentor to copy and paste. Maybe something like this:

```
- [ ] promote api staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript
 - [ ] go through the merged jakarta.ee specification website page to verify all the links are valid
 - [ ] if XML Schemas are published on https://jakarta.ee/schemas, send a PR to update the status from `Draft` to `Final`
 - [ ] approve the compatibility request
 - [ ] merge any final release branch as appropriate for the branch management for the project
```